src/TerminalSettings/TerminalSettings/MainPage.xaml

```diff
@@ -19,24 +19,47 @@
         <NavigationView.MenuItems>
             <NavigationViewItem Icon="Globe"
                 Tag="Globals_Nav" >
-                <TextBlock Tag="Globals_Page">Globals</TextBlock>
+                <TextBlock Tag="Globals_Page">General</TextBlock>
             </NavigationViewItem>
+
+            <NavigationViewItem Tag="ColorSchemes_Nav">
+                <NavigationViewItem.Icon>
+                    <FontIcon Glyph="&#xE790;"></FontIcon>
+                </NavigationViewItem.Icon>
+                <TextBlock Tag="ColorSchemes_Page">Appearance</TextBlock>
+            </NavigationViewItem>
+
             <NavigationViewItem Tag="Profiles_Nav">
                 <NavigationViewItem.Icon>
-                    <FontIcon Glyph="&#xE756;" />
+                    <FontIcon Glyph="&#xE770;" />

```

Comment:
It'd be _nice_ if you put a comment on each of the icons for what they actually are.
```suggestion
                    <!--- Color Palette Icon --->
                    <FontIcon Glyph="&#xE770;" />
```
and similarly throughout

Reply:
Let's push this in for now and format code before pushing in to the main branch.